assets/xml/objects/object_jso.xml

```diff
@@ -1,122 +1,147 @@
 ﻿<Root>
+    <!-- Assets for Garos and the Garo Master -->
```

Comment:
Small detail, but these descriptors are split between `Assets for ...` and `Objects for ...`, did you have a preference?

Reply:
I prefer the former because it explains *what* is contained within the object. If you didn't know *anything* about the N64 Zelda games or how they worked, you might ask "what's an object?" Whereas listing what's *in* the object makes it clear, even to a total outsider, what this file's purpose is. It also works better for when we extract assets from overlays too.